packages/@react-spectrum/actiongroup/chromatic/ActionGroup.chromatic-fc.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+
+import {Compact, Default, IsDisabled, IsEmphasized, IsQuiet, StaticColorBlack, StaticColorWhite} from './ActionGroup.chromatic';
+import {Flex} from '@react-spectrum/layout';
+import React from 'react';
+
+export default {
+  title: 'ActionGroup'
+};
+
+export const All = () => (
+  <Flex gap="size-100" direction={'column'}>
+    <h2>default</h2>
+    <Default />
```

Comment:
When I suggested reusing stories, I was thinking more like https://storybook.js.org/blog/component-story-format-3-0/#spreadable-story-objects

is the issue that our current chromatic stories are all csf 2 instead of csf 3?

Reply:
I did it both ways depending on how the original story was composed. This approach uses fewer snapshots in chromatic.